README.md

```diff
@@ -1291,6 +1292,186 @@ let release () =
   decr in_use
 ```
 
+### Conditions
+
+[Eio.Condition][] allows a fiber to wait until some condition is true.
+For example:
+
+```ocaml
+module X = struct
+  (* Note: this version is not safe to share across domains! *)
+
+  let x = ref 0
+  let changed = Eio.Condition.create ()
+
+  let await_zero () =
+    while !x <> 0 do Eio.Condition.await_no_mutex changed done;
+    traceln "x is now zero"
+
+  let set v =
+    x := v;
+    Eio.Condition.broadcast changed;
+    traceln "x set to %d" v
+end
+```
+
+```ocaml
+# Eio_mock.Backend.run @@ fun () ->
+  X.set 5;
+  Fiber.both
+    (fun () ->
+       traceln "Waiting for x to be 0";
+       X.await_zero ()
+    )
+    (fun () -> X.set 0);;
++x set to 5
++Waiting for x to be 0
++x set to 0
++x is now zero
+- : unit = ()
+```
+
+Note that we need a loop in `await_zero`.
+This is needed because it's possible that another fiber might set it to zero
+and then set it to something else before the waiting fiber resumes.
+
+The above version is not safe to share across domains, because `await_zero` relies on the value of `x` not changing
+after it reads it but before `await_no_mutex` registers itself with the condition.
```

Comment:
"after it is read" maybe ?

Reply:
Yeah, too many `it`s here. I replaced it with ``after `x` is read``.